### PR TITLE
Add notice about PostgreSQL file locations

### LINKED ISSUE #3484: Location for `pg_hba.conf` is incorrect for PostgreSQL on Ubuntu 20.04
### Category

This is an issue with:

- [x] errata (error in doc content)
- [ ] new doc (requires a completely new doc)
- [ ] update (add missing or refresh existing content)
- [ ] improvement (improve docs content or format)

### Affected pages

https://docs.sensu.io/sensu-go/latest/operations/deploy-sensu/scale-event-storage/#configure-postgres

### Expected behavior

When configuring PostgreSQL on Ubuntu 20.04, I should be able to create or add content to the appropriate `pg_hba.conf` file.

### Current behavior

Currently, the docs instruct users to edit the `pg_hba.conf` at `/var/lib/pgsql/data/pg_hba.conf`. This will not work for users installing Postgres on Ubuntu 20.04

### Corrections needed

For Ubuntu, the correct location is ` /etc/postgresql/14/main/pg_hba.conf`

* OS: Ubuntu 20.04

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Adds a note at the top of the page about the config file paths used in https://docs.sensu.io/sensu-go/latest/operations/deploy-sensu/scale-event-storage, with a link to PostgreSQL docs re: file locations.

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/3484

### CODE REVIEW

**Comment by hillaryfraley** [comment]
@asachs01 Looks like we're using RHEL/CentOS paths throughout this doc, so I thought the best way to handle it is a note that covers the entire page. Wdyt?

File: content/sensu-go/6.5/operations/deploy-sensu/scale-event-storage.md [hunk 1]
```diff
@@ -123,7 +129,7 @@ With this configuration complete, Postgres will have a `sensu_events` database f
 
 By default, the Postgres user you've just added will not be able to authenticate via password, so you'll also need to make a change to the `pg_hba.conf` file.
 The required change will depend on how Sensu will connect to Postgres.
-In this case, you'll configure Postgres to allow the `sensu` user to connect to the `sensu_events` database from any host using an [md5][5]-encrypted password:
+In this case, you'll configure Postgres to allow the `sensu` user to connect to the `sensu_events` database from any host using an [md5][5]-encrypted password.
```
Comment (hillaryfraley) [inline content/sensu-go/6.5/operations/deploy-sensu/scale-event-storage.md:132]:
```suggestion
In this case, you'll configure Postgres to allow the `sensu` user to connect to the `sensu_events` database from any host using an [md5][5]-encrypted password:
```

Comment (hillaryfraley) [inline content/sensu-go/6.5/operations/deploy-sensu/scale-event-storage.md]:
```suggestion
In this case, you'll configure Postgres to allow the `sensu` user to connect to the `sensu_events` database from any host using an [md5][5]-encrypted password:
```

File: content/sensu-go/6.5/operations/deploy-sensu/scale-event-storage.md [hunk 2]
```diff
@@ -305,10 +311,7 @@ Follow the steps in this section to create and add the replication role, set str
 ### Create and add the replication role
 
 If you have administrative access to Postgres, you can create the replication role.
-
-{{% notice note %}}
-**NOTE**: Complete the steps to create and add the replication role on the **primary** Postgres host.
-{{% /notice %}}
+Complete these steps to create and add the replication role on the **primary** Postgres host.
```
Comment (hillaryfraley) [inline content/sensu-go/6.5/operations/deploy-sensu/scale-event-storage.md]:
```suggestion
{{% notice note %}}
**NOTE**: Complete the steps to create and add the replication role on the **primary** Postgres host.
{{% /notice %}}
```

File: content/sensu-go/6.5/operations/deploy-sensu/scale-event-storage.md [hunk 3]
```diff
@@ -347,9 +350,7 @@ sudo systemctl restart postgresql
 
 ### Set streaming replication configuration parameters
 
-{{% notice note %}}
-**NOTE**: Complete the steps to set streaming replication configuration parameters on the **primary** Postgres host.
-{{% /notice %}}
+Complete the following steps to set streaming replication configuration parameters on the **primary** Postgres host:
```
Comment (hillaryfraley) [inline content/sensu-go/6.5/operations/deploy-sensu/scale-event-storage.md]:
```suggestion
{{% notice note %}}
**NOTE**: Complete the steps to set streaming replication configuration parameters on the **primary** Postgres host.
{{% /notice %}}
```

File: content/sensu-go/6.5/operations/deploy-sensu/scale-event-storage.md [hunk 4]
```diff
@@ -384,9 +385,7 @@ sudo systemctl restart postgresql
 
 ### Bootstrap the standby host
 
-{{% notice note %}}
-**NOTE**: Complete the steps to bootstrap the standby host on the **standby** Postgres host.
-{{% /notice %}}
+Follow these steps to bootstrap the standby host on the **standby** Postgres host:
```
Comment (hillaryfraley) [inline content/sensu-go/6.5/operations/deploy-sensu/scale-event-storage.md:384]:
```suggestion
{{% notice note %}}
**NOTE**: Complete the steps to bootstrap the standby host on the **standby** Postgres host.
{{% /notice %}}
```

**Review by asachs01 — APPROVED** [review]
LGTM @hillaryfraley ! TY! 🙇

**Pull request merged**